src/main/java/com/cleanroommc/groovyscript/compat/mods/jei/JeiPlugin.java

```diff
@@ -59,6 +83,16 @@ public void register(IModRegistry registry) {
         // jei can't handle custom recipe classes on its own
         registry.handleRecipes(ShapedCraftingRecipe.class, recipe -> new ShapedRecipeWrapper(jeiHelpers, recipe), VanillaRecipeCategoryUid.CRAFTING);
         registry.handleRecipes(ShapelessCraftingRecipe.class, recipe -> new ShapelessRecipeWrapper<>(jeiHelpers, recipe), VanillaRecipeCategoryUid.CRAFTING);
+
+        // register in world crafting recipes
```

Comment:
could you add a few default catalysts for the IWC (eg Piston for pistonPush, TNT for explosion)?

Reply:
Fluid: Water, Lava
Explosion: TNT
Burning: Fire, Flint & Steel
Piston Push: Piston, Sticky Piston